sdk/nodejs/tests/runtime/langhost/cases/019.stack_exports/index.js

```diff
@@ -0,0 +1,21 @@
+function main() {
+    return new Promise((resolve) => {
+        resolve({
+            a: new Promise((resolve2) => {
+                resolve2({
+                    x: new Promise((resolve3) => {
+                        resolve3(99);
+                    }),
```

Comment:
you could just write `Promise.resolve(99)` to simplify this.  :)

Reply:
Sure, will do.